### DISCUSSION TITLE
Timestamp of to_dataframe and GUI not matching

### DISCUSSION THREAD

**Discussion by ASAA-Team** [danielhrisca/asammdf#1020] [Q&A] [answered]
Hi,

I'm using asammdf to read a mf4 file and parse it to a dataframe.
When I'm showing the same file in the asammdf GUI, the timestamp of the samples are not matching.
I guess the to_dataframe is normalizing the values to start at 0. I've read the documentation and tried various options like use_interpolation, interpolate_outwards_with_nan and time_from_zero but I can't reach the goal of having a timestamp that's consistent to what I'm seeing in the UI.

See my explanatory screenshot below:

![asammdf_time_start](https://github.com/danielhrisca/asammdf/assets/151970946/80a45d99-68e2-40fb-93d5-b1df17d4e6ef)

Can someone elaborate on that? Am I missing something or using an option incorrectly? When looking at the code, I realize it might have something to do with the master channel.

Any help is highly appreciated, 
Thanks!

**Comment by danielhrisca** [comment]
The data frame contains only that signal?

**Reply by ASAA-Team** [reply]
Nope, just picked one example signal. But it's moreless the same for all of them, timestamps of first channels might differ, but overall it's always something like 0.25s +/-

**Reply by danielhrisca** [reply, marked as answer]
`time_from_zero=False` should work

**Reply by ASAA-Team** [reply]
You are right, @danielhrisca `time_from_zero=False` works. I believe I have made an error or used an combination of settings that made it start at 0 before. Thank you
